src/game/Strategic/Quest_Debug_System.cc

```diff
@@ -560,7 +560,7 @@ void QuestDebugScreenInit()
 
 	gItemListBox.usItemDisplayedOnTopOfList			= 1;
 	gItemListBox.usStartIndex										= 1;
-	gItemListBox.usMaxArrayIndex								= MAXITEMS;
+	gItemListBox.usMaxArrayIndex								= MAXITEMS.inner();
```

Comment:
Do you really want MAXITEMS to be an ItemId? size_t looks like a better choice to me.